home/tests/test_models.py

```diff
@@ -465,36 +466,90 @@ def test_template_create_with_pt_language(self, mock_create_whatsapp_template):
 
 class OrderedContentSetTests(TestCase):
     def test_get_gender_none(self):
+        """
+        Ordered Content Sets without a gender selected should return None
+        """
         ordered_content_set = OrderedContentSet(name="Test Title")
         ordered_content_set.save()
         self.assertIsNone(ordered_content_set.get_gender())
 
     def test_get_gender(self):
+        """
+        Ordered Content Sets with a gender selected should return the appropriate gender
+        """
         ordered_content_set = OrderedContentSet(name="Test Title")
         ordered_content_set.profile_fields.append(("gender", "female"))
         ordered_content_set.save()
         self.assertEqual(ordered_content_set.get_gender(), "female")
 
     def test_status_draft(self):
+        """
+        Draft Ordered Content Sets should return a draft status
+        """
         ordered_content_set = OrderedContentSet(name="Test Title")
         ordered_content_set.profile_fields.append(("gender", "female"))
         ordered_content_set.save()
         ordered_content_set.unpublish()
         self.assertEqual(ordered_content_set.status(), "Draft")
 
     def test_status_live(self):
+        """
+        Live Ordered Content Sets should return a live status
+        """
         ordered_content_set = OrderedContentSet(name="Test Title")
         ordered_content_set.profile_fields.append(("gender", "female"))
         ordered_content_set.save()
         self.assertEqual(ordered_content_set.status(), "Live")
 
     def test_status_live_plus_draft(self):
+        """
+        An Ordered Content Sets that is published and being drafted should return a live and draft status
+        """
         ordered_content_set = OrderedContentSet(name="Test Title")
         ordered_content_set.save()
         ordered_content_set.profile_fields.append(("gender", "female"))
         ordered_content_set.save_revision()
         self.assertEqual(ordered_content_set.status(), "Live + Draft")
 
+    def test_get_relationship(self):
+        """
+        Ordered Content Sets with a relationship selected should return the appropriate relationship
+        """
+        ordered_content_set = OrderedContentSet(name="Test Title")
+        ordered_content_set.profile_fields.append(("relationship", "single"))
+        ordered_content_set.save()
+        self.assertEqual(ordered_content_set.get_relationship(), "single")
+
+    def test_get_age(self):
+        """
+        Ordered Content Sets with an age selected should return the choosen age
+        """
+        ordered_content_set = OrderedContentSet(name="Test Title")
+        ordered_content_set.profile_fields.append(("age", "15-18"))
+        ordered_content_set.save()
+        self.assertEqual(ordered_content_set.get_age(), "15-18")
+
+    def test_get_page(self):
+        """
+        Ordered Content Sets with an page selected should return the choosen page. We compare
+        the unique slug of a page
+        """
```

Comment:
Will they always return the first page?

Reply:
It returns a list of pages, I was just checking specifically for the first page since I only imported one.

Reply:
Perhaps a better test here would be to have multiple pages and assert that `.page()` (which is badly named, apparently) returns all their slugs in order?

We probably also want a test for the case where we don't have any pages.

Reply:
Good idea, I've added that.